src/components/shared/switcherTheme/SwitcherTheme.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+
+import { useMantineColorScheme, Switch, useMantineTheme, rem } from '@mantine/core';
+import { LuSun } from 'react-icons/lu';
+import { PiMoonStars } from 'react-icons/pi';
+
+import {
+  iconColorMoon,
+  iconColorSun,
+  iconMoonHeight,
+  iconMoonWidth,
+  iconSunHeight,
+  iconSunWidth,
+} from '../../layout/header/config';
+
+const SwitcherTheme: FC = () => {
+  const { colorScheme, toggleColorScheme } = useMantineColorScheme();
+  const theme = useMantineTheme();
+
+  const sunIcon = (
+    <LuSun
+      style={{ width: rem(iconSunWidth), height: rem(iconSunHeight) }}
```

Comment:
put it in variable and reuse `{ width: rem(iconSunWidth), height: rem(iconSunHeight) }`